### DISCUSSION TITLE
2024년 07월 회고 / 일기장이 되어버린 기술 블로그

### DISCUSSION THREAD

**Discussion by 1jeongg** [BDD-CLUB/challenge-every-week-posting#614] [5. 회고]
이번 달 회고 글 입니당!

[2024년 07월 회고](https://1jeongg.github.io/2024-07-retrospect/)

분명히 기술블로그로 시작했는데 점점 일기장이 되어가고 있네요.. 반성하겠습니다 ㅠㅠ

다음 주부터는 PNU 미니 부캠도 하니까 해당 내용 정리해서 올려보겠습니다!!!! 다음 달에는 기술글로 도배 갑니다 (●'◡'●)

**Comment by pykido** [comment]
화이팅입니다!! 5대 CS 총정리가 굉장히 기대됩니다잉
